lib/features/copy-paste/BpmnCopyPaste.js

```diff
@@ -191,3 +191,9 @@ BpmnCopyPaste.$inject = [
   'eventBus',
   'moddleCopy'
 ];
+
+// helpers //////////
+
+function isLabelLike(descriptor) {
+  return descriptor.labelTarget;
```

Comment:
Why not be explicit about the shape of the object ("must have a `labelTarget` property"):

```suggestion
  return 'labelTarget' in descriptor;
```